home/migrations/0003_auto_20190527_1523.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2 on 2019-05-27 15:23
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('home', '0002_users_email'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='users',
+            name='role',
+            field=models.CharField(max_length=15),
```

Comment:
If we want to limit a user to have one role it would probably be best to use [Field.choices](https://docs.djangoproject.com/en/2.2/ref/models/fields/#django.db.models.Field.choices) to make it an enumerable choice. `max_length=15` might be a bit limiting, most databases (including PostgreSQL) use variable length storage for all text field types so it's often best to have a higher limit.

That said, I think we'll want to allow users to have more than one role. It might make sense to figure out how to do this with [django.contrib.auth](https://docs.djangoproject.com/en/dev/topics/auth/)